data/Fetch_Updated_Data_from_ModelSEED.py

```diff
@@ -1,115 +1,160 @@
 #!/usr/bin/env python
 from urllib.request import urlopen
-import sys,json
+import json
 
 MSD_git_url = "https://raw.githubusercontent.com/ModelSEED/ModelSEEDDatabase/"
 MSD_commit = "master"
 
-##
-## Compounds
-##
-file = urlopen(MSD_git_url+MSD_commit+"/Biochemistry/compounds.json")
+#
+# Compounds
+#
+file = urlopen(MSD_git_url + MSD_commit + "/Biochemistry/compounds.json")
 compounds = json.load(file)
 
-compound_headers = ["id","abbreviation","name","formula","mass","source","structure","charge","is_core",
-                    "is_obsolete","is_cofactor","deltaG","deltaGErr","pka","pkb","aliases"]
-
-cpd_header_translation={"deltaG":"deltag",
-                        "deltaGErr":"deltagerr",
-                        "structure":"inchikey"}
-
-with open('compounds.tsv','w') as cpd_fh:
-    cpd_fh.write("\t".join(compound_headers)+"\n")
+compound_headers = [
+    "id",
+    "abbreviation",
+    "name",
+    "formula",
+    "mass",
+    "source",
+    "structure",
+    "charge",
+    "is_core",
+    "is_obsolete",
+    "is_cofactor",
+    "deltaG",
+    "deltaGErr",
+    "pka",
+    "pkb",
+    "aliases",
+]
+
+cpd_header_translation = {
+    "deltaG": "deltag",
+    "deltaGErr": "deltagerr",
+    "structure": "inchikey",
+}
+
+with open("compounds.tsv", "w") as cpd_fh:
+    cpd_fh.write("\t".join(compound_headers) + "\n")
 
     for cpd in compounds:
         line_values = list()
         for field in compound_headers:
             true_field = field
-            if(field not in cpd and field in cpd_header_translation):
+            if field not in cpd and field in cpd_header_translation:
                 true_field = cpd_header_translation[field]
 
             line_values.append(str(cpd[true_field]))
 
-        cpd_fh.write("\t".join(line_values)+"\n")
+        cpd_fh.write("\t".join(line_values) + "\n")
 cpd_fh.close()
```

Comment:
You should be good for this. The builtin `open` is a context manager, which, upon exiting the block, will trigger a context manager exit function that in this case will close `cpd_fh` for you

Reply:
OK, removed